backend/Controllers/UserController.cs

```diff
@@ -21,4 +23,59 @@ public async Task<IActionResult> GetByUserName([FromRoute] string userName)
         }
         return Ok(new UserResponse { Id = matchingUser.Id, UserName = matchingUser.UserName!, });
     }
+
+    [HttpGet("GetUserId/{userId}")]
+    public async Task<User?> GetByUserId([FromRoute] string userId)
+    {
+        var matchingUser = await _userManager.FindByIdAsync(userId);
+        if (matchingUser == null)
+        {
+            return null;
+        }
+        return matchingUser;
+    }
+
+    [HttpPost("UpdateUser")]
```

Comment:
Having this be "UpdateUser" is going to meant the url for this endpoint will be "users/UpdateUser" which feels a bit redundant - I would prefer "users/update"

Reply:
Actually, on reflection what if we change it to "users/{userId}/update"?
That way we won't need to so the `GetUserById` call below.
What do you  guys think?